assets/prototype/domain/eventEditor/containers/mutations/useUpdateDateMutation.js

```diff
@@ -29,14 +29,15 @@ const useUpdateDateMutation = ({ id = 0 }) => {
 				...(description && { description }),
 				...(name && { name }),
 				...(startDate && { startDate }),
+				...(tickets && { tickets }),
 			},
 		};
 
 		return variables;
 	};
 
-	const updateHandler = ({ endDate, description, name, startDate }) => {
-		const variables = getVariables({ endDate, description, name, startDate });
+	const updateHandler = ({ endDate, description, name, startDate, tickets }) => {
+		const variables = getVariables({ endDate, description, name, startDate, tickets });
```

Comment:
We can get rid of `getVariables`, we don't really need it.

Reply:
Yes, updated here: 
https://github.com/eventespresso/event-espresso-core/pull/1947/commits/f64fd0078aae559db86462c4c3cb2864833ea357
Thanks, good sir.